pkg/cli/debug/debug.go

```diff
@@ -446,6 +448,22 @@ func (o *DebugOptions) RunDebug() error {
 	pod.Name, pod.Namespace = fmt.Sprintf("%s-debug", generateapp.MakeSimpleName(infos[0].Name)), ns
 	o.Attach.Pod = pod
 
+	// set pod security label on the namespace as the debug pod is privileged.
```

Comment:
That's not good approach, you're updating namespace for all `oc debug` operations and that's not always recommended. `oc debug` can and should be able to be invoked for regular users, since they can invoked `oc debug mypod`, not every `oc debug` invocation is `oc debug node` like in your example. Most importantly, the `oc debug` should not modify the namespace w/o prior ack from the owner of the namespace. It should most likely fail and inform user what the right action should be, here adding correct labels.

Reply:
Thank you @soltysh! I will try to find if there is a dedicated error I can parse, or, i will simply always print out an instruction message in case of error saying something like "starting debug pod failed, error=..., please make sure you set the restricted pod security on the namespace xyz using oc ..."

Reply:
> Most importantly, the oc debug should not modify the namespace w/o prior ack from the owner of the namespace. It should most likely fail and inform user what the right action should be, here adding correct labels.

Agree.  I would go so far as to suggest that instead of suggesting a label change, you suggest the user create a dedicated namespace and give that command (`oc create namespace --dry-run | oc label -f - --dry-run name=value| oc create -f -`